### ISSUE TITLE
Possible to rewire the listener?

### ISSUE THREAD

**Issue by apobbati** [senecajs/seneca-transport#28]
Hi,

I am using Consul's KV store to keep track of a map of services and their respective patterns.
Upon starting a service, a Seneca microservice connects to the KV store and sets up its listeners accordingly by reading the latest pin information from Consul. However, things can change, services can come up and go down, so I'd like to be able to re-initialize the listeners when the service-pattern mappings change.

Is there a way to do this in Seneca? Can you think of a better way to solve this problem?
Basically, I'm avoiding to have to specify the IP addresses/hostnames and pins by hardcoding them into each service. This can quickly get cumbersome if I have many services.

What do you think?

[Trello Card](https://trello.com/c/3gPQkynm/37-seneca-transport-possible-to-rewire-the-listener)

**Comment by rjrodger** [comment]
Yes there is now: https://github.com/rjrodger/seneca-balance-client
(thanks to @cjihrig and @geek !)

/cc @mcdonnelldean
